CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Release channels have their own copy of this changelog:
     feature would have prevented it
   * Updated local ledger storage so that the RPC endpoint
     `getSignaturesForAddress` always returns signatures in block-inclusion order
+  * RPC's `simulate_transaction` now returns `inner_instructions` as `json`/`jsonParsed` (#34313).
```

Comment:
Since this change doesn't affect send (the other caller of the internal simulate method), I would change this to `simulateTransaction` to refer to the RPC endpoint.
Ditto camel casing `innerInstructions`